domain-service/src/main/kotlin/pet/project/app/mapper/UserResponseMapper.kt

```diff
@@ -0,0 +1,49 @@
+package pet.project.app.mapper
+
+import pet.project.app.model.domain.DomainUser
+import pet.project.internal.commonmodels.user.User
+import pet.project.internal.input.reqreply.user.AddBookToUsersWishListResponse
+import pet.project.internal.input.reqreply.user.CreateUserResponse
+import pet.project.internal.input.reqreply.user.DeleteUserByIdResponse
+import pet.project.internal.input.reqreply.user.FindUserByIdResponse
+import pet.project.internal.input.reqreply.user.UpdateUserResponse
+
+object UserResponseMapper {
+
+    fun DomainUser.toCreateUserResponse(): CreateUserResponse {
+        return CreateUserResponse.newBuilder().also {
+            it.successBuilder.user = this.toProto()
+        }.build()
+    }
+
+    fun toAddBookToUserWishListResponse(): AddBookToUsersWishListResponse {
+        return AddBookToUsersWishListResponse.newBuilder()
+            .also { it.successBuilder }
```

Comment:
```suggestion
            .apply { successBuilder }
```

keep same code style, in this class I see different styles for same cases